server/models/video/video-live.ts

```diff
@@ -66,6 +83,27 @@ export class VideoLiveModel extends Model<Partial<AttributesOnly<VideoLiveModel>
   })
   Video: VideoModel
 
+  @ForeignKey(() => VideoLiveReplaySettingModel)
+  @Column
+  replaySettingId: number
+
+  @BelongsTo(() => VideoLiveReplaySettingModel, {
+    foreignKey: {
+      allowNull: true
+    },
+    onDelete: 'set null'
+  })
+  ReplaySetting: VideoLiveReplaySettingModel
+
+  @BeforeDestroy
+  static deleteReplaySetting (instance: VideoLiveModel) {
```

Comment:
Also add this hook in live session model, even if (for now) we never remove them